### PR TITLE
Address IESG comments

### PR DESCRIPTION
**Pull request by DavidSchinazi**
This PR attempts to address comments received from the IESG.

### CODE REVIEW

File: charter.md [hunk 1]
```diff
@@ -23,15 +23,15 @@ The WebTransport working group will define new client-server protocols
...
+The WebTransport working group will define an application-layer protocol or suite of
+application-layer protocols that support a range of simple communication methods.
 These must include unreliable messages (that might be limited by
 the path MTU), reliable messages, and ordered streams of reliable
 messages.  Attention will be paid to the performance of the protocol,
 in particular protocol overheads and the potential for head-of-line
 blocking; its ability to be deployed and used reliably under different
 network conditions; and the ability to integrate protocol use into
-the Web security model.
+the Web security model. These new protocols shall be built over QUIC or TLS.
```
Comment (martinthomson) [inline charter.md]: I would instead say that the working group won't define new transport protocols.

Comment (aboba) [reply charter.md]: "The WG will not define new transport protocols but will instead use existing protocols such as QUIC and TLS."

File: charter.md [hunk 2]
```diff
@@ -23,15 +23,15 @@ The WebTransport working group will define new client-server protocols
 or protocol extensions in order to support the development of the
 W3C WebTransport API <https://wicg.github.io/web-transport>.
 
-The WebTransport working group will define a protocol or suite of
-protocols that support a range of simple communication methods.
+The WebTransport working group will define an application-layer protocol or suite of
+application-layer protocols that support a range of simple communication methods.
```
Comment (mirjak) [inline charter.md]: What's about e.g. saying ".. define one or multiple bindings to provide the following simple communication methods on top of QUIC, HTTP/3, and/or TLS/TCP"...?

Comment (DavidSchinazi) [reply charter.md]: I'm personally not a fan of the word "binding" as it is more vague than "application-layer protocol"

File: charter.md (hunk 1 above)
Comment (DavidSchinazi) [reply charter.md]: I like that. Applied.

**Pull request merged**